host-bin/startxiwi

```diff
@@ -0,0 +1,93 @@
+#!/bin/sh -e
+# Copyright (c) 2015 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+set -e
+
+APPLICATION="${0##*/}"
+OPTS_ENTER=''
+OPTS_XIWI=''
+USAGE="$APPLICATION [options] CHROOT_APP [options]
+
+Wraps enter-chroot to launch a xiwi-app in a new windowed or tabbed session.
+You can launch any individual chroot graphical application in crouton windows.
+
+By default, it will log into the primary user on the first chroot found.
+
+Enter '-h' for 'enter-chroot' options and enter '-H' for 'xiwi-app' options.
+"
+USAGE_ENTER="enter-chroot [options]
+
+  -b          Fork and run the specified command silently in the background.
+  -c CHROOTS  Directory the chroots are in. Default: /mnt/stateful_partition/crouton/chroots
+  -l          Make the command part of a login. Parameters are passed directly
+              to the chroot command, and a call to su is appended.
+  -k KEYFILE  Override the auto-detected encryption key location.
+  -n NAME     Name of the chroot to enter. Default: first one found in CHROOTS
+  -t TARGET   Only enter the chroot if it contains the specified TARGET.
+  -u USERNAME Username (or UID) to log into. Default: 1000 (the primary user)
+  -X XMETHOD  (Override the auto-detected XMETHOD for this session.) - deprecated, 'xiwi' is enforced.
+  -x          Does not log in, but directly executes the command instead.
+              Note that the environment will be empty (sans TERM).
+              Specify -x a second time to run the /prepare.sh script.
+"
+USAGE_XIWI="xiwi [options] CHROOT_APP [options]
```

Comment:
I'm not sure of a good way to deduplicate the help text, but most (if not all) of the text in `xiwi`'s help text probably should be here.